dags/copy_deduplicate.py

```diff
@@ -168,25 +168,41 @@
     )
     baseline_args = [
         "--project-id=moz-fx-data-shared-prod",
-        "--date={{ ds }}",
         "--only=*_stable.baseline_v1",
+        "--start_date={{ ds }}",
+        "--end_date={{ ds }}"
     ]
     baseline_clients_first_seen = GKEPodOperator(
         task_id="baseline_clients_first_seen",
         name="baseline-clients-first-seen",
-        arguments=["script/run_glean_baseline_clients_first_seen"] + baseline_args,
+        command=[
+            "bqetl",
+            "query",
+            "backfill",
+            "*.baseline_clients_first_seen_v1",
+        ] + baseline_args,
```

Comment:
This seems to work for https://github.com/mozilla/telemetry-airflow/blob/2407ba20bf5e6fc35fd5ab3d3194d6d7302bc5a1/dags/stripe.py#L20